src/pages/data-management/index.jsx

```diff
@@ -82,14 +75,16 @@ const DataManagementPage = ({ route }) => {
 
     // Right now we have one experiment per project, so we can just load the experiment
     // This has to be changed when we have more than one experiment
-    const activeExperimentId = activeProject.experiments[0];
+    const activeExperimentId = projectsList[activeProjectUuid].experiments[0];
+
+    dispatch(loadProcessingSettings(activeExperimentId));
```

Comment:
How does this prevent the issue at hand? What triggers the loading of backup status in here?

Reply:
This PR removes `samples` from this `useEffect`'s dependency array. When `samples` is updated, the call to `loadBackendStatus` is called.  `samples` contain keys that track the upload progress, and it is continuously updated when uploading a sample, resulting in the continuous calls to `loadBackendStatus`.